src/tests/class_tests/openms/source/SvmTheoreticalSpectrumGenerator_test.cpp

```diff
@@ -95,7 +96,16 @@ START_SECTION(void simulate(PeakSpectrum &spectrum, const AASequence &peptide, b
   PeakMap exp;
   MzMLFile mz_file;
 
+#if OPENMS_BOOST_VERSION_MINOR < 56
   mz_file.load(OPENMS_GET_TEST_DATA_PATH("SvmTheoreticalSpectrumGenerator_test.mzML"),exp);
+  TEST_EQUAL(spec.size(), 7);
+#else
+  mz_file.load(OPENMS_GET_TEST_DATA_PATH("SvmTheoreticalSpectrumGenerator_test_boost58.mzML"),exp);
+  TEST_EQUAL(spec.size(), 8);
+  // the extra peak:
+  TEST_EQUAL(spec.getStringDataArrays()[0][2], "YIon  0++") // TODO: ion_nr is always zero, its actually y4++
+  TEST_EQUAL(spec.getIntegerDataArrays()[0][2], 2)
+#endif
```

Comment:
why does changing the RNG produce additional peak in this model? @timosachsenberg maybe you can comment on this why that could be the case?

we are simulating the peptide IFSQVGK? We now get an additional ion at 216.134 which is a y4++ ion that does not appear with the previous boost...seems a bit scary if the result not just numerically but qualitatively (e.g. number of peaks) depends so much on the RNG